### ISSUE TITLE
[Term]: schema:name

### ISSUE THREAD

**Issue by ben-norton** [DiSSCo/openDS#186]
### Term Name

schema:name

### Digital Object Name

Agent

### Feedback

In schema.org, name has a much wider scope. There it refers to any identifiable entity. Here, name has a narrower scope, the preferred name of the Agent. Therefore, there is not an exact match between the terms and the property in openDS should be changed to the openDS namespace.

**Comment by DavidFichtmueller** [comment]
I think that this term is fitting, as Agent is a really generic class. Besides a person, it could also represent a software system or an organisation, so having a property with a wide scope is fitting. An alternative would be to use [foaf:name](http://xmlns.com/foaf/spec/#term_name) which also has a really wide definition: "A name for some thing."

**Comment by wouteraddink** [comment]
We have discussed this, and concluded that if for every term we want to reuse we need an exact match, we can almost not reuse any terms, which does not help interoperability. Also this is not how schema.org is currently used. It is intended to be very broad so it can be used for most purposes. Foaf:name could also be used, but we have opted for schema. org as we can reuse more terms from it than from foaf so it limits the number of namespaces we need to include.
